scripts/MAC Season 5.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "MAC Season 5 Metadata Upload to bety"
+output: html_document
+---
+

```

Comment:
link to appropriate documentation, i.e. say "see [link] for information on setting up a database clone or connecting to terraref betydb. For example, 

`docker run --name bety -p 5432:5432 terraref/bety-postgis`